x/feegrant/keeper/grpc_query.go

```diff
@@ -110,20 +110,23 @@ func (q Keeper) AllowancesByGranter(c context.Context, req *feegrant.QueryAllowa
 	var grants []*feegrant.Grant
 
 	store := ctx.KVStore(q.storeKey)
-	pageRes, err := query.Paginate(store, req.Pagination, func(key []byte, value []byte) error {
-		var grant feegrant.Grant
-
-		granter, _ := feegrant.ParseAddressesFromFeeAllowanceKey(key)
+	prefixStore := prefix.NewStore(store, feegrant.FeeAllowanceKeyPrefix)
```

Comment:
I am unfamiliar with this addition while cherry-picking. It comes from #10830 and is a feature from 0.46.

Reply:
No, this looks correct!

We need to iterate over the `FeeAllowanceKeyPrefix` prefix. Looks like the key is composed as follows: `FeeAllowanceKeyPrefix | grantee | granter`